docs/examples/Rocket_Car.rst

```diff
@@ -0,0 +1,1585 @@
+##
+
+.. raw:: html
+
+   <center>
+
+:math:`\mathbf{\text{Modeling of a Variable-Mass Nonholonomic Gyrostatic Rocket Car Using Extended Kane's Equations}}`
+
+.. raw:: html
+
+   </center>
```

Comment:
Headings need to be convereted to standard RST like the other examples. This should be:

```rst
============================================================================
Modeling of a Variable-Mass Nonholonomic Gyrostatic Rocket Car Using Extended Kane's Equations
=============================================================================
```

Reply:
Also, we shouldn't copy the title of the journal paper if that is what you've done here. Make your own title for this example and cite the other paper as the inspiration.

Reply:
Use this heading level method to match the other files:

```rst
=============
Title of page
=============
Heading
=======

Subheading
----------